lua/onenord/theme.lua

```diff
@@ -551,8 +551,25 @@ local function load_plugins(config)
     CmpItemAbbrDeprecated = { fg = onenord.fg },
     CmpItemAbbrMatch = { fg = onenord.blue, style = "bold" },
     CmpItemAbbrMatchFuzzy = { fg = onenord.blue, underline = true },
-    CmpItemKind = { fg = onenord.purple },
-    CmpItemMenu = { fg = onenord.light_gray },
```

Comment:
I think you accidentally deleted this, can you put it back? My suggestion had it moved since I wanted to group the `CmpItemKind` groups together.